live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -25,20 +25,23 @@
   when: lookup('env', 'APPLIANCE_PASSWORD') == ''
   no_log: true
 
-#
-# While the "delphix-platform" package will take care of creating the
-# delphix user, it won't configure the user's password. Thus, we have to
-# do that here, during the intial build of the appliance.
-#
+- file:
```

Comment:
We should follow this up by removing the duplicate code from delphix-platform

Reply:
Yea, I'll do this after landing.